durabletask/client.py

```diff
@@ -97,6 +97,7 @@ def __init__(self, *,
                  log_handler: Optional[logging.Handler] = None,
                  log_formatter: Optional[logging.Formatter] = None,
                  secure_channel: bool = False):
+        self._metadata = metadata
```

Comment:
Is this change needed? Seems like we're not using `_metadata` anywhere in this file.